### PR TITLE
CBL-2491: Update info plist - Copyright & CBLEdition

### PR DESCRIPTION
**Pull request by jayahariv**
* make sure, copyright always uses the current year when building new binary
* also pass CBLEdition property with the info plist to distinguish between EE and CE

### CODE REVIEW

File: Objective-C/Info.plist [hunk 1]
```diff
@@ -19,8 +19,10 @@
 	<key>CFBundleVersion</key>
 	<string>$(CBL_BUILD_NUMBER)</string>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © 2017 Couchbase. All rights reserved.</string>
+	<string>$(CBL_COPYRIGHT)</string>
```
Comment (pasin) [inline Objective-C/Info.plist:22]: I'm a bit reluctant to this change and it seems like it's safe to have the copy string in the Info.plist incase somebody grab the project and build by themselves without using the script.

Comment (pasin) [reply Objective-C/Info.plist:22]: Can we do two things?
1. < string >Copyright © $(CBL_COPYRIGHT_YEAR) Couchbase. All rights reserved.< / string>
2. Add a build phase script to assign a default value to CBL_COPYRIGHT_YEAR variable if it doesn't have  the value? (NEED to try it out)

**Review by pasin — CHANGES_REQUESTED** [review]

File: xcconfigs/Project.xcconfig [hunk 2]
```diff
@@ -21,6 +21,7 @@ CBL_VERSION_STRING                 = 3.1.0
 CBL_BUILD_NUMBER                   = 0
 CBL_EXPORTED_SYMBOLS_FILE          = Objective-C/CouchbaseLite.exp
 CBL_SWIFT_MODULEMAP_FILE           = Swift/CouchbaseLiteSwift.modulemap
+CBL_COPYRIGHT_YEAR                 = 2020
```
Comment (pasin) [inline xcconfigs/Project.xcconfig:24]: Should it be 2021?

**Review by pasin — APPROVED** [review]
One comment about the year. The rest looks good to me.

File: xcconfigs/Project.xcconfig (hunk 2 above)
Comment (jayahariv) [reply xcconfigs/Project.xcconfig:24]: I kept that for testing, forget to revert it.
```suggestion
CBL_COPYRIGHT_YEAR                 = 2021
```

**Pull request merged**